### PR TITLE
Flex gap proper release

### PR DESCRIPTION
**Pull request by argyleink**
Restoring article with minor edits to tense. 
Confirmed release for chrome 85, let's launch the article same day 👍 

cc @davidsgrogan

### CODE REVIEW

**Comment by davidsgrogan** [comment]
Looks good. This is coming in Chrome 84 though, not 85.

**Comment by argyleink** [comment]
nice, ok, removed it! 
there's no chrome 84 label tho..?

**Comment by kaycebasques** [comment]
@davidsgrogan & @argyleink what day exactly do you want this to go live?

File: src/site/content/en/blog/flexbox-gap/index.md [hunk 1]
```diff
@@ -1,21 +1,234 @@
...
-Please excuse our dirty laundry. You are seeing this page so that links are not
-broken. 
+### Browser compatibility
 
-A technical error has caused this article to be published before it was clear
-which version of Chrome is shipping Flexbox `gap`. We expect it to land
-sometime in the next few versions. Watch this space for details.
-{% endBanner %}
+At the time of writing `gap` is supported in desktop Firefox 63, Firefox for Android 63, 
+and for Chromium-based browsers as of 85. See
```
Comment (kaycebasques) [inline src/site/content/en/blog/flexbox-gap/index.md:17]:
```suggestion
and Chromium 85 (desktop and Android). See
```

File: src/site/content/en/blog/flexbox-gap/index.md [hunk 2]
```diff
@@ -1,21 +1,234 @@
...
+
+{% Aside %}
+The ownership of the spacing shifts from the child to the parent
+{% endAside %}
+
+In the first 2 examples (without Flexbox `gap`), the children are targeted and 
+assigned spacing from other elements. In the antidote gap example, the container 
+owns the spacing. Each child can relieve itself of the burden, while also 
+centralizing the spacing ownership. Simplifying consistency. Reorder, 
+change viewports, remove elements, append new elements, etc and spacing remains
```
Comment (kaycebasques) [inline src/site/content/en/blog/flexbox-gap/index.md:182]:
```suggestion
change viewports, remove elements, append new elements, etc. and spacing remains 
```

File: src/site/content/en/blog/flexbox-gap/index.md [hunk 3]
```diff
@@ -1,21 +1,234 @@
...
+{% endAside %}
+
+In the first 2 examples (without Flexbox `gap`), the children are targeted and 
+assigned spacing from other elements. In the antidote gap example, the container 
+owns the spacing. Each child can relieve itself of the burden, while also 
+centralizing the spacing ownership. Simplifying consistency. Reorder, 
+change viewports, remove elements, append new elements, etc and spacing remains 
+consistent. No new selectors, no new media queries, just space.
+
+## Chromium DevTools Updates
```
Comment (kaycebasques) [inline src/site/content/en/blog/flexbox-gap/index.md:185]:
```suggestion
## Chromium DevTools updates

```

File: src/site/content/en/blog/flexbox-gap/index.md [hunk 4]
```diff
@@ -1,21 +1,234 @@
...
+With these updates come changes to Chromium DevTools, notice how the **Styles** 
+pane handles `grid-gap` and `gap` now 👍
+
+<figure class="w-figure">
+  <img src="./devtools-gap.png" alt="An office with two people working at a table." style="border-radius: 10px;" width="400">
+  <figcaption class="w-figcaption">Devtools shows the both grid-gap and gap, with 
+  gap shown used below grid-gap as to let the cascade use the latest syntax.</figcaption>
```
Comment (kaycebasques) [inline src/site/content/en/blog/flexbox-gap/index.md:192]:
```suggestion
  <figcaption class="w-figcaption">Devtools shows the both <code>grid-gap</code> and <code>gap</code>, with 
  <code>gap</code> shown used below <code>grid-gap</code> as to let the cascade use the latest syntax.</figcaption>
```

File: src/site/content/en/blog/flexbox-gap/index.md [hunk 5]
```diff
@@ -1,21 +1,234 @@
...
+</figure>
+
+DevTools supports both `grid-gap` and `gap`, this is because `gap` is essentially 
+an alias to the previous syntaxes. 
+
+
+## New layout potential
+
+With Flexbox `gap`, we unlock more than convenience, we unlock powerful, perfectly
```
Comment (kaycebasques) [inline src/site/content/en/blog/flexbox-gap/index.md:201]:
```suggestion
With Flexbox `gap`, we unlock more than convenience. We unlock powerful, perfectly 
```

**Comment by argyleink** [comment]
I think July 14th is the date, @davidsgrogan can you confirm?

File: src/site/content/en/blog/flexbox-gap/index.md [hunk 6]
```diff
@@ -1,21 +1,235 @@
...
-Please excuse our dirty laundry. You are seeing this page so that links are not
-broken. 
+### Browser compatibility
 
-A technical error has caused this article to be published before it was clear
-which version of Chrome is shipping Flexbox `gap`. We expect it to land
-sometime in the next few versions. Watch this space for details.
-{% endBanner %}
+At the time of writing `gap` is supported in desktop Firefox 63, Firefox for Android 63, 
+and Chromium 85 (desktop and Android). See
```
Comment (davidsgrogan) [inline src/site/content/en/blog/flexbox-gap/index.md:17]:
```suggestion
and Chromium 84 (desktop and Android). See
```

**Comment by davidsgrogan** [comment]
I had previously said July 14th, M84's estimated release date.

But, instead, maybe it can go live whenever because the new functionality exists in 84 not behind any flag. That was not true when the article first went live.

So I think it can actually go live as soon as @argyleink makes the 85->84 change I suggested in https://github.com/GoogleChrome/web.dev/pull/3284#pullrequestreview-436307355 (apologies if I didn't do that right, I am inept with github review tools).

WDYT?

**Comment by argyleink** [comment]
ya did great!

**Pull request merged**